filewriter_v2/README.md

```diff
@@ -0,0 +1,390 @@
+| [slow_pixel_direction](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-detector-detector-module-slow-pixel-direction-field) | NX_NUMBER | Size of a pixel along the slowest varying pixel direction. |
+| [slow_pixel_direction@depends_on](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-detector-detector-module-slow-pixel-direction-depends-on-attribute) | NX_CHAR | `"/entry/instrument/detector/transformations/translation"` |
+| [slow_pixel_direction@offset](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-detector-detector-module-slow-pixel-direction-offset-attribute) | NX_NUMBER[3] | A fixed offset applied before the transformation. |
+| [slow_pixel_direction@transformation_type](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-detector-detector-module-slow-pixel-direction-transformation-type-attribute) | NX_CHAR | `"translation"` |
+| slow_pixel_direction@units | NX_LENGTH | `"m"` |
+| [slow_pixel_direction@vector](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-detector-detector-module-slow-pixel-direction-vector-attribute) | NX_NUMBER[3] | Unit vector describing the slowest varying pixel direction. |
+
+## [/entry/instrument/detector/transformations](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-detector-transformations-group)
+
+| Name | NeXus Type | Description |
+|---|---|---|
+|  |  |
+| @NX_class | NX_CHAR | [`"NXtransformations"`](https://manual.nexusformat.org/classes/base_classes/NXtransformations.html#nxtransformations) |
+| translation | NX_FLOAT | Distance from the sample to the detector. |
+| translation@depends_on | NX_CHAR | `"."` |
+| translation@offset | NX_NUMBER[3] | A fixed offset applied before the transformation. |
+| translation@transformation_type | NX_CHAR | `"translation"` |
+| translation@units | NX_LENGTH | `"m"` |
+| translation@vector | NX_NUMBER[3] | Unit vector describing the direction of the translation from the sample to the detector. |
+| two_theta | NX_FLOAT[nP] | Starting $2\theta$ value for each frame. |
+| two_theta@offset | NX_NUMBER[3] | A fixed offset applied before the transformation. |
+| two_theta@transformation_type | NX_CHAR | `"rotation"` |
+| two_theta@units | NX_ANGLE | `"degree"` |
+| two_theta@vector | NX_NUMBER[3] | Unit vector describing the axis of the rotation. |
+| two_theta_end | NX_FLOAT[nP] | $2\theta$ end value for each frame. |
+| two_theta_end@units | NX_ANGLE | `"degree"` |
+| two_theta_increment_set | NX_FLOAT | The intended average range through which $2\theta$ moves during the exposure of a frame. |
+| two_theta_increment_set@units | NX_ANGLE | `"degree"` |
+
+
+
+## [/entry/sample]((https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-sample-group))
```

Comment:
Broken link (extra parentheses).

Reply:
Fixed in [32d64b2](https://github.com/soph-dec/documentation/pull/1/commits/32d64b26e8e47ee5a550d2c323a7966b39b922a5).